skymodel.py

```diff
@@ -0,0 +1,208 @@
+        Must be in range [300.0,30000.0] and < wmax
+        (default is 300.0)
+    wmax: float
+        Maximum wavelength (nm) in the wavelength grid.
+        Must be in range [300.0,30000.0] and > wmin
+        (default is 2000.0)
+    wgrid_mode: str
+        Mode of the wavelength grid.
+        options: ['fixed_spectral_resolution','fixed_wavelength_step', 'user']
+        (default is 'fixed_wavelength_step')
+    wdelta: float
+        Wavelength sampling step dlam in range [0,30000.0] (nm/step)
+        (default is 0.1)
+    wres: int
+        lam/dlam where dlam is wavelength step.
+        In range [0,1.0e6] (default is 20000)
+    wgrid_user: list of floats
+        default is [500.0, 510.0, 520.0, 530.0, 540.0, 550.0]
+    lsf_type: str
+        Line spread function type for convolution.
+        options: ['none','Gaussian','Boxcar'] (default is 'none')
+    lsf_gauss_fwhm: float
+        Gaussian full-width half-max for line spread function wavelength bins.
+        Range > 0.0 (default is 5.0)
+    lsf_boxcar_fwhm: float
+        Boxcar full-width half-max for line spread function wavelength bins.
+        Range > 0.0 (default is 5.0)
+
+    observatory: str
+        Observatory where observation takes place.
+        Options are 'paranal', 'lasilla', 'armazones' (default is 'paranal')
+
+    Returns
+    -------
+    trans_waves, transmission: tuple of arrays of floats
+        'trans_waves' is an array of wavelengths in angstroms (float),
+        'transmission' is an array of fractional atmospheric
+        transmittance (float).
+
+    """
+
+    params = locals()
+
+    if params['observatory'] == 'lasilla':
+        params['observatory'] = '2400'
+    elif params['observatory'] == 'paranal':
+        params['observatory'] = '2640'
+    elif (params['observatory'] == '3060m' or
+          params['observatory'] == 'armazones'):
+        params['observatory'] = '3060'
+    else:
+        raise ValueError('Wrong Observatory name, please refer to the '
+                         'skycalc_cli documentation.')
+
+    # Use the bit from skycalc_cli which queries from the SkyCalc Sky Model
+    server = 'http://etimecalret-001.eso.org'
+    url = server + '/observing/etc/api/skycalc'
+    response = requests.post(url, data=json.dumps(params))
+    results = json.loads(response.text)
+
+    status = results['status']
+    tmpdir = results['tmpdir']
+    tmpurl = server + '/observing/etc/tmp/' + tmpdir + '/skytable.fits'
+
+    if status == 'success':
+        try:
+            response = requests.get(tmpurl, stream=True)
+            data = response.content
+        except requests.exceptions.RequestException as e:
+            print(e, 'could not retrieve FITS data from server')
+    else:
+        raise ParameterError('HTML request failed. A custom parameter you ' +
+                             'set is most likely not accepted by the SkyCalc' +
+                             ' Calculator. json.loads() returns: ', results)
```

Comment:
Error message is one string.
```suggestion
                             ' Calculator. json.loads() returns: {}'.format(results))
```